deploy/059_deploy_lens_spokepool.ts

```diff
@@ -24,7 +24,14 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     hubPool.address,
     hubPool.address,
   ];
-  const constructorArgs = [WGHO[spokeChainId], QUOTE_TIME_BUFFER, FILL_DEADLINE_BUFFER];
+  const l2USDCBridge = "0x7188B6975EeC82ae914b6eC7AC32b3c9a18b2c81";
```

Comment:
nb. move to consts.

Reply:
https://github.com/across-protocol/contracts/pull/941/commits/a14915f4e5af4f378cb904b8ab5a17a667efe133